src/main/java/com/google/sps/data/Accessory.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+public class Accessory {
+
+  public enum Type {
+    HAT,
+    GLASSES,
+    COMPANION
+  };
+
+  private String id;
+  private String imageFilePath;
+  private Type type;
+  private int height;
+  private int width;
+  private int xPos;
+  private int yPos;
+
+  public Accessory(
+      String id, String imageFilePath, Type type, int height, int width, int xPos, int yPos) {
+    this.id = id;
+    this.imageFilePath = imageFilePath;
+    this.type = type;
+    this.height = height;
+    this.width = width;
+    this.xPos = xPos;
+    this.yPos = yPos;
+  }
+
+  public String getId() {
+    return this.id;
+  }
+
+  public String getImageFilePath() {
+    return this.imageFilePath;
+  }
+
+  public Type getType() {
+    return this.type;
+  }
+
+  public int getHeight() {
+    return this.height;
+  }
+
+  public int getWidth() {
+    return this.width;
+  }
+
+  public int getXPos() {
+    return this.xPos;
+  }
+
+  public int getYPos() {
+    return this.yPos;
+  }
+}
```

Comment:
Are there setters as well or will they be final with construction?

Reply:
+1

Reply:
I haven't had a need for setters yet. I don't think I will need them since I think these accessories will be static data finalized upon construction. I will add setters if needed.